plugins/crate-copy-azure/src/main/java/io/crate/copy/azure/AzureFileOutput.java

```diff
@@ -52,7 +57,7 @@ public AzureFileOutput(URI uri, SharedAsyncExecutor sharedAsyncExecutor, Setting
 
     @Override
     public OutputStream acquireOutputStream(Executor executor, WriterProjection.CompressionType compressionType) throws IOException {
-        OutputStream outputStream = operator.createOutputStream(resourcePath);
+        OutputStream outputStream = operator.createOutputStream(resourcePath, MAX_BYTES);
```

Comment:
Not adding any tests since it's basically third party code and it's tested in https://github.com/apache/opendal/pull/5422